devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy

```diff
@@ -176,11 +186,20 @@ def poll() {
  */
 def refresh() {
 	log.trace "ZigBee DTH - Executing refresh() for device ${device.displayName}"
-	def cmds =
+	
+	if (device.getDataValue("model") == "E261-KR0B0Z0-HA") {
+		def cmds = []
```

Comment:
just move `def cmds = []` and the `return` outside the if/else

Reply:
ok